ONSdigital/eq-author
# Upgrade eslint-config-eq-author

### What is the context of this PR?
 - Upgrades `eslint-config-eq-author` which reduces severity level of `no-unused-vars` to warn
 - Moves to `devDependencies`
  
### How to review 
- Run tests
- Ensure you can add unused variables or parameters and still hot load
- Ensure `lint` warns you of unused variables or parameters
